src/common/i18n.tsx

```diff
@@ -107,10 +107,8 @@ export function I18nProvider(props: RenderableProps<I18nProviderProps>) {
     }, [isLoading, props.locale]);
   }
 
-  const value = useMemo(
-    () => ({ t, setLocale, locale: props.locale }),
-    [t, setLocale, props.locale]
-  );
+  const langTag = useMemo(() => t('lang_tag'), [t, props.locale])
```

Comment:
Nit: GitHub is showing a bunch of prettier errors (missing semicolon in this case). I wonder if you can run prettier on your branch or set up your editor to run it automatically on save?

Reply:
Yeah, sorry about all the CI failure emails. What's weird is I thought I saw it prettier running as a pre-commit hook, but I guess that was just ESLint?

Reply:
Looks like I only configured it to run on `.ts` files (not `.tsx` or `.json` files etc.)

https://github.com/birchill/10ten-ja-reader/blob/9a0ce777f8a17070094523c329f2162ff92d9dd2/package.json#L55-L58

Reply:
Ah, that makes sense. I can change it as part of this PR.

Reply:
Okay, well I did run the `lint_staged` task before pushing up my latest commit (also updated it to include `.tsx` files), but I didn't think about the couple of style errors from previous commits that weren't in the staged files, so the pipeline still sent a failure message for that push...but hopefully the last one 😄.